clusterctl/cmd/root.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"flag"
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/apiserver/pkg/util/logs"
+	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/cluster-api/util"
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "clusterctl",
+	Short: "cluster management",
+	Long:  `Simple kubernetes cluster management`,
+	Run: func(cmd *cobra.Command, args []string) {
+		// Do Stuff Here
+		cmd.Help()
+	},
+}
+
+var (
```

Comment:
On my previous comment on flags, I do mean all flags on all commands should not be implemented yet. They can be implemented when there is actually functionality to back it. This way the usage and description is accurate to how it is actually used.

Reply:
The var seems to still be here. There also still seem to be flags on some of the commands that are not backed by implementation.

Reply:
Weird my linter didn't complain.